context.cc

```diff
@@ -27,11 +28,12 @@ ContextPtr NewContext(IsolatePtr iso,
   // side to lookup the context in the context registry. We use slot 1 as slot 0
   // has special meaning for the Chrome debugger.
   Local<Context> local_ctx = Context::New(iso, nullptr, global_template);
-  local_ctx->SetEmbedderData(1, Integer::New(iso, ref));
 
   m_ctx* ctx = new m_ctx;
   ctx->ptr.Reset(iso, local_ctx);
   ctx->iso = iso;
+  local_ctx->SetEmbedderData(1, Integer::New(iso, ref));
```

Comment:
Make the indices an enum so we know which values are in use.

A note for myself that `ref` here should become a Handle, as per the updated https://go.dev/wiki/cgo#function-variables